integrationExamples/mass/index.html

```diff
@@ -0,0 +1,107 @@
+<html>
+  <head>
+      <script>
+        window.massConfig = {
+          inskin: {
+            default: {
+              plr_PubfileId: '000000/mass'
+            }
+          }
+        };
+
+        var PREBID_TIMEOUT = 3300;
+  
+        var adUnits = [{
+          code: 'div-gpt-ad-1460505748561-0',
+          mediaTypes: {
+            banner: {
+              sizes: [[300, 250], [300, 600]],
+            }
+          },
+          bids: [{
+            bidder: 'ix',
+            params: {
+              siteId: '123456',
+              size: [300, 250]
+            }
+          }]
+        }];
+  
+        var pbjs = pbjs || {};
+        pbjs.que = pbjs.que || [];
+      </script>
+
+      <script>
+        pbjs.que.push(function() {
+          pbjs.setConfig({
+            mass: {
+              enabled: true
+            }
+          });
+        });
+      </script>
+  
+      <script type="text/javascript" src="/build/dist/prebid.js" async></script>
```

Comment:
Is this working on your local after you execute `gulp serve`?  
I had to change `src` to `../../build/dev/prebid.js` on my local to load pbjs properly.

Reply:
@FilipStamenkovic Thanks for this, it's now fixed. I also added a notice to let people know that in order for the page to actually work, you need access to a bid simulation tool.